src/store/ConfigStore.ts

```diff
@@ -43,7 +43,7 @@ const namespacePersistFilter = whitelistInputWithInitialState(
 const persistConfig = {
   key: persistKey,
   storage: storage,
-  whitelist: ['authentication', 'namespaces', 'serverConfig'],
+  whitelist: ['authentication', 'namespaces', 'serverConfig', 'jaegerState'],
```

Comment:
Should we store all the ´jaegerState´? I'm worried about storing information that might not be uptated the next time we use Kiali such as: toolbar.isFetchingService and jaegerURL.

Please, if needed, consider to only store a subset of this information, see: https://github.com/kiali/kiali-ui/blob/master/src/store/ConfigStore.ts#L30-L41

Reply:
Totally agree , I am still testing why the state of redux take time in get updated before the rendering of the component, We'll only persist the jaegerUrl.